src/output.rs

```diff
@@ -67,7 +67,21 @@ fn print_entry_colorized(
             .map(Style::to_ansi_term_style)
             .unwrap_or(default_style);
 
-        write!(stdout, "{}", style.paint(component.to_string_lossy()))?;
+        let path_string = component.to_string_lossy();
+
+        match config.path_separator {
+            None => write!(stdout, "{}", style.paint(path_string))?,
+            Some(sep) => {
```

Comment:
A couple of things:
* This should probably be in a separate function
* I don't think this is the best way to solve this (convert to byte vector, loop over vector, convert to string). Why not use a string-replacement function?
* Instead of checking for `cfg!(windows)`, this code should replace `std::path::MAIN_SEPARATOR` with `config.path_separator`. Otherwise, on Windows, you would replace both `/` and `\` with the new separator.

Reply:
If we are to use a string replacement, we can also handle any string replacement; it would make the code simpler. Do you have anything against it?